File: mycollab-services/src/main/java/com/esofthead/mycollab/common/InvalidTokenException.java
package com.esofthead.mycollab.common;

/**
 * 
 * @author MyCollab Ltd.
 * @since 4.0
 * 
 */
public class InvalidTokenException extends Exception {
	private static final long serialVersionUID = 1L;

	public InvalidTokenException(Throwable e) {
		super(e);
	}

	public InvalidTokenException(String message) {
		super(message);
	}
}

File: mycollab-services/src/main/java/com/mycollab/common/InvalidTokenException.kt
package com.mycollab.common

import com.mycollab.core.MyCollabException

/**
 * @author MyCollab Ltd.
 * @since 4.0
 */
class InvalidTokenException(message: String) : MyCollabException(message) {
    companion object {
        private val serialVersionUID = 1L
    }
}
